apps/verticalhome/js/clock.js

```diff
@@ -70,6 +61,10 @@ function Clock() {
   this.handleEvent = function cl_handleEvents(e){
 
     switch(e.type) {
+      /**
+       * This event is thrown when time format has been changed.
```

Comment:
is it just me or it's pretty common that word 'thrown' is associated with the exceptions and 'sent'/'received' are associated with events? I personally would prefer to use 'received' here.